config/database.yml

```diff
@@ -23,7 +23,7 @@ default: &default
 
 development:
   <<: *default
-  database: forms_api
+  database: forms_api_development
```

Comment:
❤️ , however we might have to change some dockerfile config to ensure this doesn't break docker builds

Reply:
Ah, good catch. I looked at forms_deploy/local just now, it looks like the configuration is already slightly different from what is in the Rails config files:

```
forms-deploy on  main at 09:52:13
% git grep forms_admin local
local/docker-compose.yml:      - DATABASE_URL=postgres://forms_admin:forms_admin@postgres/forms_admin
local/postgres/initdb.d/init.sql:CREATE USER forms_admin WITH PASSWORD 'forms_admin' CREATEDB;
local/postgres/initdb.d/init.sql:CREATE DATABASE forms_admin;
local/postgres/initdb.d/init.sql:GRANT ALL PRIVILEGES ON DATABASE forms_admin TO forms_admin;
local/postgres/initdb.d/init.sql:ALTER DATABASE forms_admin OWNER TO forms_admin;
```

So I don't think this PR will necessarily make things any worse, but we should probably make the change in forms_deploy to make things even more consistent.